gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFSInputStream.java

```diff
@@ -38,6 +39,8 @@
 import org.apache.hadoop.fs.statistics.IOStatistics;
 import org.apache.hadoop.fs.statistics.IOStatisticsSource;
 
+@VisibleForTesting
```

Comment:
I don't think we should annotate the class but the constructor along with comment (as in your previous implementation).

Reply:
removed